README.md

```diff
@@ -2,7 +2,11 @@
 [![Jenkins Plugin](https://img.shields.io/jenkins/plugin/v/pipeline-input-step.svg)](https://plugins.jenkins.io/pipeline-input-step)
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/pipeline-input-step.svg?color=blue)](https://plugins.jenkins.io/pipeline-input-step)
 
-Adds the Pipeline `input` step to wait for human input or approval.
+Adds the Pipeline `input` step to wait for human input or approval. 
+
+For further understanding,check the https://plugins.jenkins.io[documentation] for pipeline-input-step plugin.  The link:https://www.jenkins.io/redirect/pipeline-snippet-generator [Pipeline Syntax Snippet Generator] guides the user to select checkout options.
```

Comment:
Use markdown format instead of asciidoc format for the hyperlink markup.
```suggestion
For further understanding,check the [documentation](https://plugins.jenkins.io) for pipeline-input-step plugin.  The  [Pipeline Syntax Snippet Generator](https://www.jenkins.io/redirect/pipeline-snippet-generator) guides the user to select checkout options.
```